samples/tensorflow/magic_wand/train/requirements.txt

```diff
@@ -0,0 +1,2 @@
+numpy==1.16.2
+tensorflow==2.0.0-beta1
```

Comment:
2.4.1? or latest?

Reply:
I checked the upstream and on 2.4.1, it is 2.4.1... so there's a chance that the copy I adapted on this commit doesn't match the original on 2.4.1 upstream. Thanks for pointing this out. I'll be checking this over soon.

Reply:
So, after checking upstream and checking out the 2.4.1 release, I found out I was wrong and the tensorflow package version number upstream for the 2.4.1 release is 2.0.0-beta. Since I have some problems running the converted model following the `train/README.md` after upgrading the `requirements.txt` for tensorflow 2.4.0 (and numpy 1.19.2), I think it might be best to leave those alone for the time being:

![image](https://user-images.githubusercontent.com/14157493/115773707-3e341080-a376-11eb-8220-44470b45cdf2.png)

Do you have thoughts?

Details: [The file in the 2.4.1 release](https://github.com/tensorflow/tensorflow/blob/85c8b2a817f95a3e979ecd1ed95bff1dc1335cff/tensorflow/lite/micro/examples/magic_wand/train/requirements.txt) has "2.0.0-beta1" in it. My confusion came from [the file's history on the main branch](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/micro/examples/magic_wand/train/requirements.txt) changing the version number to 2.4.0 being backdated to December even though the release came out in January.